hail/python/hail/gg/geoms.py

```diff
@@ -0,0 +1,501 @@
+import abc
+import plotly
+
+from .aes import aes
+from .utils import categorical_strings_to_colors, continuous_nums_to_colors
+import hail as hl
+
+from ..ir.utils import is_continuous_type
+
+
+class FigureAttribute:
+    pass
+
+
+class Geom(FigureAttribute):
+
+    def __init__(self, aes):
+        self.aes = aes
+
+    @abc.abstractmethod
```

Comment:
If you subclass abc.ABC you get some sanity checks:
```
In [1]: import abc                                                                                                                                                                                                                             

In [2]: class A: 
   ...:     @abc.abstractmethod 
   ...:     def foo(): 
   ...:         pass 
   ...:                                                                                                                                                                                                                                        

In [3]: A()                                                                                                                                                                                                                                    
Out[3]: <__main__.A at 0x7fe48736ab90>

In [4]: class B(abc.ABC): 
   ...:     @abc.abstractmethod 
   ...:     def foo(): 
   ...:         pass 
   ...:                                                                                                                                                                                                                                        

In [5]: B()                                                                                                                                                                                                                                    
Traceback (most recent call last):
  File "<ipython-input-5-a9cb5187b7ad>", line 1, in <module>
    B()
TypeError: Can't instantiate abstract class B with abstract methods foo

```

Reply:
Good call, will do.